test/integration.sh

```diff
@@ -122,6 +122,23 @@ else
   exit 1
 fi
 
+rm -rf $CL_HOME/dropins
+mkdir -p $CL_HOME/dropins
+cp -R $SCRIPT_DIR/packages-src/bonjour $CL_HOME/dropins
+
+echo "> test the LOG_LEVEL command"
```

Comment:
There are two tests with the same name: test the the LOG_LEVEL command

Reply:
```suggestion
echo "> test the LOG_LEVEL command - no log level from configuration"
```